ontology/current-version/RiC-O_1.1.rdf

```diff
@@ -7524,6 +7564,34 @@
       <RiCCMCorrespondingComponent xml:lang="en">RiC-R026 ('has provenance'
             relation)</RiCCMCorrespondingComponent>
    </owl:ObjectProperty>
+   <!-- https://www.ica.org/standards/RiC/ontology#hasDateAssociatedWithOrganicProvenance -->
```

Comment:
or maybe hasOrganicProvenanceDate (same, I am splitting hairs, but IMHP it is simpler and sounds better)